### ISSUE TITLE
IDE Fatal error after closing project in PyCharm

### ISSUE THREAD

**Issue by john-todorovic** [sourcery-ai/sourcery#171]
### Issue description or question

After closing the project in PyCharm, and then opening a different one, I got a window pop-up notification with the following stacktrace:

```
com.intellij.serviceContainer.AlreadyDisposedException: Already disposed: Project(name=Image_Handler, containerState=DISPOSE_COMPLETED, componentStore=D:\Python\Image_Handler) (disposed)
	at com.intellij.serviceContainer.ComponentManagerImpl.checkState(ComponentManagerImpl.kt:187)
	at com.intellij.serviceContainer.ComponentManagerImpl.getComponent(ComponentManagerImpl.kt:537)
	at com.intellij.codeInsight.daemon.DaemonCodeAnalyzer.getInstance(DaemonCodeAnalyzer.java:24)
	at ai.sourcery.intellij.statemachine.SourceryStateMachine.lambda$handleLogin$4(SourceryStateMachine.java:258)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:265)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

### Sourcery Version

0.9.5

### Code editor or IDE name and version

PyCharm 2021.2.3 CE

### OS name and version

Windows 10 Enterprise 21H1

**Comment by Hellebore** [comment]
I can reproduce this - now looking into a fix

**Comment by Hellebore** [comment]
Fixed with 0.10.3
